src/pages/recurringExpenses/index/RecurringExpenses.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { date } from 'common/helpers';
+import { useCurrentCompanyDateFormats } from 'common/hooks/useCurrentCompanyDateFormats';
+import { DataTable, DataTableColumns } from 'components/DataTable';
+import { EntityStatus } from 'components/EntityStatus';
+import { Default } from 'components/layouts/Default';
+import { useTranslation } from 'react-i18next';
+
+export function RecurringExpenses() {
+  const [t] = useTranslation();
+  const { dateFormat } = useCurrentCompanyDateFormats();
+  const pages = [
+    { name: t('recurring_expenses'), href: '/recurring_expenses' },
+  ];
+
+  const columns: DataTableColumns = [
+    {
+      //no status id enum
+      id: 'status_id',
+      label: t('status'),
+    },
+    {
+      id: 'number',
+      label: t('number'),
+    },
+    { id: 'vendor_id', label: t('vendor') },
+
+    { id: 'client_id', label: t('client') },
+    //no frequency id enum
```

Comment:
I could not find status id enum in
https://github.com/invoiceninja/invoiceninja/blob/v5-develop/app/Models/RecurringExpense.php
so i couldn't make status badge.